django/core/checks/settings_deprecation.py

```diff
@@ -0,0 +1,87 @@
+from django.conf import settings
+from django.core import checks
+
+
+def deprecated_settings_variables(*args, **kwargs):
+    """
+    This check warns users who still use deprecated settings variables.
+    """
+
+    deprecated_settings_list = (
+        # todo: add older settings
+        # Django 1.4
+        "TRANSACTIONS_MANAGED",
+        # Django 1.5
+        "SITE_ID",
```

Comment:
I avoid `django.contrib.sites` personally, but I'm sure it's not deprecated.

Reply:
True. Removed it.